### ISSUE TITLE
Installation Failed, what is Default User Name for VM Login

### ISSUE THREAD

**Issue by cybrpimp** [jiannelli/vagrant-redmine#1]
I tried to build this box tonight and it did not work.  In trying to figure out what went wrong I tried to log in to the box but did not have the default user credentials.

**Comment by jiannelli** [comment]
You should be able to login into the box by using the command `vagrant ssh`, and then you can use `sudo` if you need to. https://www.vagrantup.com/docs/cli/ssh.html

**Comment by cybrpimp** [comment]
@jiannelli Thanks for your reply.  I was able to get into the box using `vagrant ssh`.

I took a look around but was not able to figure out why the installation did not work and Redmine does not come up on the address: http://172.16.27.196

I have watched the build and do not see any specific errors so I'm not sure where to start.

Do you have any ideas where to start to track down the problem?

**Comment by jiannelli** [comment]
When you do `vagrant up` it should start upgrading and downloading packages and showing you progress.  After a while it should finish.

Maybe you got a network configuration problem with your host and guest and they do not see each other (try if you can ping from the host and from the guest)

**Comment by cybrpimp** [comment]
My other boxes worked just fine so I'm thinking the network is ok.  One issue that I an curious about is the 172.16.27.196 address.

I wonder if I change that to a more standard 192.168 address if that might help?

Would it mess up anything else in your configuration?

**Comment by jiannelli** [comment]
Yes, try the ip adersses that works for you and let me know how it worked.
